osu.Game.Rulesets.Rush/Beatmaps/RushGeneratedBeatmapConverter.cs

```diff
@@ -323,14 +323,16 @@ private LanedHit createNormalHit(HitObject original, LanedHitLane lane, IList<Hi
                 Samples = original.Samples
             };
 
-        private StarSheet createStarSheet(HitObject original, LanedHitLane lane, IList<HitSampleInfo> samples) =>
-            new StarSheet
+        private StarSheet createStarSheet(HitObject original, LanedHitLane lane, IList<HitSampleInfo> samples)
+        {
+            return new StarSheet
```

Comment:
Any reason for removing the expression syntax?

Reply:
No. I just changed it for easier tests, and forgot to change it back. I'll fix that.